cypress/support/e2e.ts

```diff
@@ -65,6 +65,13 @@ beforeEach(() => {
     res.headers['origin'] = 'http://localhost:3000'
     res.continue()
   })
+
+  // Graphql security policies are based on Origin headers.
+  // These are stripped by cypress because chromeWebSecurity === false; this adds them back in.
+  cy.intercept('https://api.uniswap.org/v1/graphql', (res) => {
+    res.headers['origin'] = 'http://localhost:3000'
```

Comment:
we should use `Origin=https://app.uniswap.org` to match `fetch-schema.js` does this work with cypress?

Reply:
optionally, instead of intercepting, can you see if adding that field to the `headers` param in `src/graphql/data/RelayEnvironment.ts`

Reply:
I switched the origin, also I tried adding to relay as well and it seems it does not work